README.md

```diff
@@ -1,21 +1,21 @@
-# k8s-devkit Helm Charts
+# cisco-sso helm charts
```

Comment:
Readme has accidentally changed the repo name

Reply:
@dcwangmit01  this was intentional.  I think we should rename this git repository to `charts` and relate it back to the github org (cisco-sso) rather than k8s-devkit.